src/main/java/org/example/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private int addCalledalledCount = 0;
+
+    public int getAddCalledalledCount() {
+        return addCalledalledCount;
+    }
+
+    public int getCalledCount() {
+        return addCalledalledCount;
+    }
+
+    public ArrayList<String> getDelimiterAndNumberString(String numbers) {
```

Comment:
**getDelimiterAndNumberString** method doing two things here you can write separate methods which does single task at a time